app/src/main/java/org/coepi/android/repo/SymptomRepo.kt

```diff
@@ -20,6 +20,7 @@ interface SymptomRepo {
 
 class SymptomRepoImpl(reportRepo: TcnReportRepo): SymptomRepo {
 
+    // TODO: localize strings
```

Comment:
You can solve it in this PR if you want. Inject `Resources` to retrieve strings in the view model.

Reply:
@i-schuetz so would I just inject the Resources class into this SymptomsRepo since we are retrieving the strings here instead of in the VM?